README.md

```diff
@@ -1,2 +1,24 @@
-# hedera-sdk-cpp
-C++ SDK for Hedera
+# Hedera™ C++ SDK
+
+> The SDK for interacting with Hedera Hashgraph: the official distributed
+> consensus platform built using the hashgraph consensus algorithm for fast,
+> fair and secure transactions. Hedera enables and empowers developers to
+> build an entirely new class of decentralized applications.
+
+## SDK IS NOT READY FOR PRODUCTION USE. IT IS CURRENTLY STILL UNDER DEVELOPEMENT. 
+
+## Contributing
+
+Contributions are welcome. Please see the
+[contributing guide](https://github.com/hashgraph/.github/blob/main/CONTRIBUTING.md)
+to see how you can get involved.
+
+## Code of Conduct
+
+This project is governed by the
+[Contributor Covenant Code of Conduct](https://github.com/hashgraph/.github/blob/main/CODE_OF_CONDUCT.md). By
+participating, you are expected to uphold this code of conduct.
+
+## License
+
+[Apache License 2.0](LICENSE)
```

Comment:
I'd like to clone the repo and try out the build -- should we put the commands in here for how to build and test?

Reply:
Yeah I can work on adding that today